### ISSUE TITLE
Default value datagrid doesn't work

### ISSUE THREAD

**Issue by souhirBen** [orocrm/platform#404]
Hi,

I'm trying to use a default value in my datagrid but it doesn't work. It works only after I click in options -> discard changes (the link next to the Title).

```yml
datagrid:
    example-grid:
        filters:
            default:
                datetime-field:
                    value:
                        start: '2015-02-01'
                        end: '{{2}}' 
                     type: %oro_filter.form.type.filter.datetime_range.class%::TYPE_BETWEEN

```

**Comment by sergeyz** [comment]
Hi, @souhirBen !

please try next keys

```
                    value:
                        date_start: '2015-02-01'
                        date_end: '{{2}}' 
```

from https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/FilterBundle/Filter/AbstractDateFilter.php#L37-L38

**Comment by souhirBen** [comment]
Hi @sergeyz  I tried it doesn't  work. I tried also with string value to be sure it doesn't work.

**Comment by sergeyz** [comment]
I can confirm that default filter is not applied, thanks for your report!

**Comment by lmeliberty** [comment]
Hello, 

Do you have news to fix this problem ? 
I've the same problem and it's not possible for the momment to update my project for my client.

Thanks

**Comment by iklymenko** [comment]
Hi @lmeliberty,
since we have already several concerns related to this issue we will work on during next 2 weeks and include fix in the closest release. Thanks for your patience.

**Comment by lmeliberty** [comment]
Hi @iklymenko,

Great, thanks for your reply.

**Comment by sergeyz** [comment]
Hi! Fixed in 1.7-1.9 branches and master https://github.com/orocrm/platform/commit/0156e977fe0abb8b57717489408448788024a4e9
